thrift/history.thrift

```diff
@@ -139,6 +139,7 @@ struct RespondDecisionTaskCompletedRequest {
 
 struct RespondDecisionTaskCompletedResponse {
   10: optional RecordDecisionTaskStartedResponse startedResponse
+  20: optional bool requestActivityStartEnabled
```

Comment:
ActivitiesToDispatchLocally[]?

Reply:
This should be a list or map, so that server can update the list if it want to.
So basically Client provides the initial list, and server has the right to update it (based on other info/criteria tbd).

Reply:
Will that be over complicated? how could server know/control which activity should be directly start?